src/main/webapp/service-worker.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Welcome to your Workbox-powered service worker!
+ *
+ * You'll need to register this file in your web app and you should
+ * disable HTTP caching for this file too.
+ * See https://goo.gl/nhQhGp
+ *
+ * The rest of the code is auto-generated. Please don't update this file
+ * directly; instead, make changes to your Workbox build configuration
+ * and re-run your build process.
+ * See https://goo.gl/2aRDsh
+ */
+
+importScripts("https://storage.googleapis.com/workbox-cdn/releases/4.3.1/workbox-sw.js");
```

Comment:
Single quotes please (everywhere in this file).

Reply:
Oh this file was generated I believe when I build the React application (I didn't create it myself). So I'm assuming whenever I build/deploy my site, it will still end up having the double quotes.
Do you still want me to change it @zkoogle